### DISCUSSION TITLE
Cannot Rename Folders By Pressing Enter

### DISCUSSION THREAD

**Discussion by ProjectMoon** [open-webui/open-webui#6391] [Issues]
# Bug Report

## Installation Method

Docker (cuda tag)

## Environment

- **Open WebUI Version:** 0.3.33

- **Operating System:** Gentoo Linux
- **Browser (if applicable):** Firefox 131

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [X] I have included the browser console logs.
- [X] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Pressing enter after typing in a new folder name renames the folder.

## Actual Behavior:

Pressing enter ends the editing of the folder name, but folder name remains unchanged.

## Description

**Bug Summary:**
You can change the name of the folder by clicking outside of the folder name while editing the name.

## Reproduction Details

**Steps to Reproduce:**
1. Create folder.
2. Click side menu and pick rename.
3. Type new name.
4. Hit enter.

## Logs and Screenshots

**Browser Console Logs:**
The only thing that comes in the browser console is:
```
Edit [RecursiveFolder.svelte:285:10 (RecursiveFolder.svelte)]
```
This happens when you click on the rename button in the side menu.

**Comment by tjbck** [comment]
Cannot reproduce, keep us updated!

**Reply by ProjectMoon** [reply]
Did you try it in Firefox? I tried it in Chromium, and it worked. But it doesn't work in Firefox. But no JS errors in console.

**Comment by ProjectMoon** [comment]
So this does not happen in Chromium. But it does happen on Firefox desktop and Firefox mobile, so I suspect it's a bug specific to Firefox.  I even tried doing it with addons/strict tracking protection off and it didn't work.

**Reply by tjbck** [reply]
PR Welcome!

**Comment by silentoplayz** [comment]
If you're using [v0.3.34](https://github.com/open-webui/open-webui/releases/tag/v0.3.34) or the latest version of Open WebUI, this issue should be fixed for Firefox browser.
> 🖥️ Folder Rename Fix for Firefox: Addressed a persistent issue where users could not rename folders by pressing enter in Firefox, now ensuring seamless folder management across browsers.
